client/src/main/java/io/getstream/chat/android/client/call/RetrofitCall.kt

```diff
@@ -35,6 +35,8 @@ internal class RetrofitCall<T : Any>(
         call.cancel()
     }
 
+    fun originalCall(): retrofit2.Call<T> = call
```

Comment:
Nit: could be exposed as just the property itself, not sure we need this separate getter for it.